questions/108-medium-trim/template.ts

```diff
@@ -1 +1,6 @@
-export type Trim<S extends string> = any
+export type TrimLeft<S extends string> = S extends ` ${infer Rest}` ? TrimLeft<Rest> : S;
```

Comment:
Should you export the helper types?

Reply:
No, because in this particular case it is not intended to be used outside the module, my bad 😅